images/win/scripts/ImageHelpers/ChocoHelpers.ps1

```diff
@@ -29,4 +29,21 @@ function Choco-Install {
             }
         }
     }
+}
+
+function Get-ChocoPackageLatestVersion {
+    [CmdletBinding()]
+    param(
+        [Parameter(Mandatory)]
+        [string] $PackageName,
+        [Parameter(Mandatory)]
+        [string] $TargetVersion

```

Comment:
maybe not Target but Major since this function is intended to work with major version if I get it right
```suggestion
        [string] $MajorVersion
```